### ISSUE TITLE
Engine - Replace seal between saildrive and hull

### ISSUE THREAD

**Issue by smr547** [CanberraOceanRacingClub/namadgi3#816]
Chris Green and I were just chatting about what other big expenses might be due for an aging yacht, so I decided to check the manual

According to the [Volvo Service Protocol](https://github.com/CanberraOceanRacingClub/namadgi3/files/14808266/Volvo_ServiceProtocol_D2_75F.pdf)
 replacement is required every 7 years. I'll seek a quote from Boatworks. This could be expensive as it usually requires an engine lift to remove the saildrive.

When researching this item for Trilogy we found comments on yachting forums  reporting when the owners did this maintenance they found the seal to be looking "as new". 

This might be a difficult decision but lets wait for quote. Definitely something to inject into the budgeting process.

[Volvo_ServiceProtocol_D2_75F.pdf](https://github.com/CanberraOceanRacingClub/namadgi3/files/14808266/Volvo_ServiceProtocol_D2_75F.pdf)

#budget=$7000

@pcbaston @Christopher-w-green @CORCpete @allister-polkinghorne  will be interested

**Comment by pcbaston** [comment]
This issue is related to the Saildrive Seal Replacement.

As all manufacturers do, they propose periodic maintenance based on an
item's projected life.
Being a seal, it is subject of degradation over time. However, INMO, most
seals outlive their 'expected' service life.
Seals usually only degrade due to:

   1. physical damage (tear, etc)
   2. prolonged exposure to environment that changes the physical
   properties of the seal (hardens, softens, breaks down)

The second mechanism of seal degradation usually results in small leaks at
first, which gets worse with use and time, until catastrophic failure
occurs.
I recently replaced my 2-+ yo BMW sump gasket due to a slow leak that has
been present for 5+ years.

My recommendation would be to:

   - Note the service requirement
   - Plan for the expenditure
   - Not perform the S-drive seal replacement unless absolutely necessary
   (water leak, sail drive damage, etc) in favour of replacement when the
   engine is removed for overhaul

Note - I discussed Engine Replacement recently with Steve Ring and
recommended Engine Rebuild (~$10,000-15,000) over replacement (~$30,000)
when required.

Note - I am not a mechanical engineer or engine mechanic, but am
experienced on engines and seals/gaskets. I studied Materials Engineering
as part of my undergrad degree.  I do not seek to contradict the
recommendations of a qualified materials engineer.

Regards,

*Paul Baston*
